addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -67,6 +68,7 @@ export class LinkPopover extends Component {
                     ?.pop() ||
                 "",
             isImage: this.props.isImage,
+            isLabelEditable: !this.props.linkElement.childElementCount,
```

Comment:
maybe `isLabelHidden: !!this.props.linkElement.childElementCount` then we can remove the `!` in the template, easier to understand